wait-for-node.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env sh
 
-until $(curl --output /dev/null --silent --head --fail http://localhost:8080/health); do
+until $(curl --output /dev/null --silent --head --fail https://test-ocn.emobilify.com/health); do
```

Comment:
It should be taken for envs. It's not possible to setup app on many envs when we have hardcoded links

Reply:
TO be changed after it start working well